### DISCUSSION TITLE
Old projects do not open with the new version

### DISCUSSION THREAD

**Discussion by Hoopeman** [DISTRHO/Cardinal#227] [General]
As the title says, all the projects that I had with the old version do not load! Cardinal opens with the new configuration but not a trace of my constructions is as if it did not detect them I use cubase 11 mac osx 10.3 Sierra

**Comment by falkTX** [comment]
Which exact old version? what plugin format? Might be expected if you used VST3, there was a bug where state was not being saved properly before.

**Comment by Hoopeman** [comment]
I couldn't tell you which version since I installed on top of it and the installer is confusing, it says something like "Cardinal-macOS-universal-15b6ca03" anyway, whenever I opened the old version with Cubase it didn't give me any kind of problem, the builds disappeared from this version, thanks

**Comment by falkTX** [comment]
I dont understand what you are saying, but ok.

is the issue there with 22.04 tagged release, or only with the nightly builds (github actions)?

**Comment by Hoopeman** [comment]
It never gave me a problem! not even with the dark version, it has been with this new update

**Comment by Hoopeman** [comment]
thanks for the work! Is there no way I can get my jobs back? I installed the old version and it doesn't work either! Thanks in advance

**Comment by falkTX** [comment]
I have no idea what the issue is or what is going on, so for now, no.
feel free to investigate though

**Comment by Hoopeman** [comment]
Thanks mate
